test/unit/routes/test-overview.js

```diff
@@ -18,11 +18,15 @@ const VALID_URL_WITHOUT_OVERVIEW = '/' + workloadTypes.PROBATION + '/offender-ma
 
 const OM_MISSING_ID_URL = '/' + workloadTypes.PROBATION + '/offender-manager/overview'
 
-const OM_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/offender-manager/1/overview/csv'
-const TEAM_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/team/1/overview/csv'
-const LDU_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/ldu/1/overview/csv'
-const REGION_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/region/1/overview/csv'
-const HMPPS_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/hmpps/0/overview/csv'
+const OM_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/offender-manager/1/overview/caseload-csv'
+const TEAM_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/team/1/overview/caseload-csv'
+const LDU_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/ldu/1/overview/caseload-csv'
+const REGION_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/region/1/overview/caseload-csv'
+const HMPPS_OVERVIEW_CSV_URL = '/' + workloadTypes.PROBATION + '/hmpps/0/overview/caseload-csv'
+
+const TEAM_REDUCTIONS_CSV_URL = '/' + workloadTypes.PROBATION + '/team/1/overview/reductions-csv'
+const LDU_REDUCTIONS_CSV_URL = '/' + workloadTypes.PROBATION + '/ldu/1/overview/reductions-csv'
+const REGION_REDUCTIONS_CSV_URL = '/' + workloadTypes.PROBATION + '/region/1/overview/reductions-csv'
```

Comment:
Maybe add a test to check for the correct error when hit for `hmpps` and `offender-manager` level